tcms/core/views.py

```diff
@@ -78,6 +78,26 @@ def get_context_data(self, **kwargs):
                 ),
             )
 
+        # Check for SSL usage
+        doc_url = (
+            "https://kiwitcms.readthedocs.io/en/latest/"
+            "installing_docker.html#ssl-configuration"
+        )
+        if not self.request.is_secure():
+            messages.add_message(
+                self.request,
+                messages.ERROR,
+                mark_safe(  # nosec:B308:B703
+                    _(
+                        "You are not using secure connection (SSL). "
+                        'See <a href="%(doc_url)s">documentation</a>'
+                    )
+                    % {
+                        "doc_url": doc_url,
+                    }
+                ),
+            )
+
```

Comment:
See `tests/test_http.sh` and add additional checks for the error message with http/https.

Reply:
ping ^^^